packages/adapters/ethers/src/client.ts

```diff
@@ -346,7 +347,8 @@ export class EthersAdapter extends AdapterBlueprint {
 
     if (type === 'AUTH') {
       const { address } = await (selectedProvider as unknown as W3mFrameProvider).connect({
-        chainId
+        chainId,
+        preferredAccountType: OptionsController.state.defaultAccountTypes.eip155
```

Comment:
Do we already setting the `OptionsController` before this line is getting called? Might there be race condition that we are initializing adapters before initializing the controllers? Lets just double check, otherwise if possible we could directly use this value from props rather than state?